sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyWrapUnwrapRequest.java

```diff
@@ -24,6 +24,24 @@
     @JsonProperty(value = "value", required = true)
     private Base64Url value;
 
+    /**
+     * Initialization vector for symmetric algorithms.
+     */
+    @JsonProperty(value = "iv")
+    private byte[] initializationVector;
```

Comment:
These other parameters "iv", "aad", and "tag" aren't used for key wrap or unwrap. See the public API for .NET at https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Keys/api/Azure.Security.KeyVault.Keys.netstandard2.0.cs. We can certainly discuss changes, but what we have now are designed to lead the customer to use the right parameters and get only the data useful back in a request.